drizzlepac/haputils/align_utils.py

```diff
@@ -1222,9 +1222,14 @@ def update_image_wcs_info(tweakwcs_output, headerlet_filenames=None, fit_label=N
                     wcs_name = '{}-FIT_{}_{}'.format(wname, fit_label, item.meta['fit_info']['catalog'])
 
             # establish correct mapping to the science extensions
-            sci_ext_dict = {}
-            for sci_ext_ctr in range(1, num_sci_ext + 1):
-                sci_ext_dict["{}".format(sci_ext_ctr)] = fileutil.findExtname(hdulist, 'sci', extver=sci_ext_ctr)
+            try:
+                sci_ext_dict = {}
+                for sci_ext_ctr in range(1, num_sci_ext + 1):
+                    sci_ext_dict["{}".format(sci_ext_ctr)] = fileutil.findExtname(hdulist, 'sci', extver=sci_ext_ctr)
+            except Exception:
+                exc_type, exc_value, exc_tb = sys.exc_info()
+                traceback.print_exception(exc_type, exc_value, exc_tb, file=sys.stdout)
+                logging.exception("message")
```

Comment:
I am sure I don't follow everything that is going on here.  Is 'logging.exception' reporting the exception with the explanation of 'message'?  If so, should 'message' be replaced with something more explanatory?  Just checking...

Reply:
The code looks misleading, but here is a simple example to show what would be printed.  The "traceback.print_exception() generates the first 3 lines.  The logging.exception("message") generates the output starting from "ERROR:root:message".  In this instance (perhaps in all instances), it is not necessary to have both "print" statements.

import sys, traceback, logging
try:
    1/0
except Exception:
    exc_type, exc_value, exc_tb = sys.exc_info()
    traceback.print_exception(exc_type, exc_value, exc_tb, file=sys.stdout)
    logging.exception("message")

Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ZeroDivisionError: division by zero
ERROR:root:message
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ZeroDivisionError: division by `zero``

Reply:
Sounds good to me.